lms/templates/ccx/schedule.html

```diff
@@ -72,26 +73,28 @@
     <h2>${_('Schedule a Unit')}</h2>
     <form role="form" id="add-unit" name="add-unit" class="ccx-form">
       <div class="field">
-        <b>${_('Section')}</b><br/>
-        <select name="chapter"></select>
+        <label for="ccx_chapter"><b>${_('Section')}</b></label>
+        <select name="chapter" id="ccx_chapter"></select>
       </div>
       <div class="field">
-        <b>${_('Subsection')}</b><br/>
-        <select name="sequential"></select>
+        <label for="ccx_sequential"><b>${_('Subsection')}</b></label>
```

Comment:
Please provide a classname for this label. Something like `form-label` will work, but make sure the styles look good (if applicable).